data/json/monsters/zed-animal.json

```diff
@@ -482,6 +482,48 @@
       "PUSH_VEH"
     ]
   },
+  {
+    "id": "mon_zllama",
+    "type": "MONSTER",
+    "name": { "str_sp": "vicious vicuña" },
```

Comment:
Should add a comment for translators about the origin of the name

Reply:
It's done. See line 489.